libs/core/src/lib/utils/interfaces/css-style-builder.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { HasElementRef } from '../public_api';
+
+export interface Hash {
```

Comment:
Maybe lets move Hash interface to separate file?
I think I'ts going to be used widely in application, and it would be better to import it from '../hash' than '../css-style-builder'

Or make hash with more generic approach as:
```
export interface Hash <keyType, valueType> {
    [key: keyType]: valueType;
}
```

And use it like `Hash<string, any>`

Reply:
key
![image](https://user-images.githubusercontent.com/10849982/71722532-72a25700-2e29-11ea-9949-e46b4aaac9a5.png)

Reply:
key have to be string or number, interface has been moved to another file with generic value type